extensions/ql-vscode/test/unit-tests/packages/commands/CommandManager.test.ts

```diff
@@ -0,0 +1,8 @@
+import { CommandManager } from "../../../../src/packages/commands";
+
+describe(CommandManager.name, () => {
+  it("can create a command manager", () => {
+    const commandManager = new CommandManager();
+    expect(commandManager).not.toBeUndefined();
```

Comment:
I agree we probably want a test file for this, but I would argue this is actually testing Typescript functionality rather than something specific to the `CommandManager`. 

We can probably overwrite it later with some more concrete testing (and maybe that's what you were thinking as well?).

Reply:
Yes, exactly, this was meant as a test just to have a test, but doesn't actually test anything. When we actually do add tests, we'll remove this test.